torch/_inductor/autotune_process.py

```diff
@@ -646,19 +656,19 @@ def make_run_fn(
         if "warmup" in inspect.signature(run_method).parameters:
             warmup_arg["warmup"] = False
 
-        from torch._C import _cuda_getCurrentRawStream as get_raw_stream
-
-        if torch.version.hip and self.matrix_instr_nonkdim != 0:
+        if output_tensor.device.type == "cpu":
```

Comment:
?

Reply:
The two branches of the if/else were entirely identical, so I just repurposed one of them

Reply:
though I suppose it would be cleaner to factor out the `functools.partial` call and just use the branch for the value of `stream`